metaflow/plugins/argo/argo_workflows.py

```diff
@@ -401,6 +402,14 @@ def _process_parameters(self):
                 )
             seen.add(norm)
 
+            if param.kwargs.get("type") == JSONType or isinstance(
+                param.kwargs.get("type"), FilePathClass
```

Comment:
@savingoyal should I use `isinstance()` for both comparisons?  For some reason I didn't fully understand, only the `FilePathClass` required the use of it and it failed with the `==` comparison.